06-improve-model-performance/improvement-overview.ipynb

```diff
@@ -0,0 +1,913 @@
+{
```

Comment:
<p>Can you give an explanation of the text it returned? Something like "we asked the agent about bonds and it pulled the following text from the document X.pdf".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ContextualAI/examples/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>